packages/admin/cms-admin/src/userPermissions/user/UserPageToolbar.tsx

```diff
@@ -38,7 +44,25 @@ export const UserPermissionsUserPageToolbar = ({ userId }: { userId: string }) =
             </ToolbarTitleItem>
             <ToolbarFillSpace />
             <ToolbarActions>
-                <StartImpersonationButton userId={userId} />
+                <CrudMoreActionsMenu
+                    overallActions={[
+                        isAllowed("impersonation")
+                            ? {
+                                  label: <FormattedMessage id="comet.impersonate" defaultMessage="Impersonate" />,
```

Comment:
Maybe this should be added to [messages.ts](https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/messages.ts), then this only needs to be defined once and can also be used in `UserGrid.tsx` and maybe other places.

Reply:
efd661ea3813416bdeddd8f44be5da33b38afc34